### ISSUE TITLE
nghttpx:  route request and strip path prefix 

### ISSUE THREAD

**Issue by inokcom** [nghttp2/nghttp2#787]
In config:
backend=localhost,8080;example.com/foo/

Now, nghttpx forward request to example.com/foo/bar/index.html to localhost:8080/foo/bar/index.html
Is it possible to strip /foo/ part and forward localhost:8080/bar/index.html?

**Comment by tatsuhiro-t** [comment]
No, currently it is not possible to strip any path component on forwarding.

**Comment by agc93** [comment]
Is there any plans to allow this? Stripping the path component used to route the request (`/foo/` in this case) would be extremely useful for proxying to more than one backend app
